.github/workflows/test.yml

```diff
@@ -163,6 +181,23 @@ jobs:
         go-version: 1.16.x
     - name: Checkout code
       uses: actions/checkout@v2
+    - name: Setup selenium standalone server
+      run: | 
+        sudo apt-get update
+        sudo apt-get install -y xvfb
+        sudo apt-get install -y google-chrome-stable
+
+        wget https://chromedriver.storage.googleapis.com/92.0.4515.107/chromedriver_linux64.zip
+        unzip chromedriver_linux64.zip
+        sudo mv -f chromedriver /usr/local/bin/chromedriver
+
+        mkdir -p cmd/selenium
+        cd cmd/selenium
+        wget  https://selenium-release.storage.googleapis.com/3.14/selenium-server-standalone-3.14.0.jar
+        mv selenium-server-standalone-3.14.0.jar selenium-server-standalone.jar
+        
+        xvfb-run -a --server-args="-screen 0 1280x1024x24" java -jar ./selenium-server-standalone.jar &
+        cd -
     - name: Setup Kubernetes
```

Comment:
Could you please move the above code to a script and call that script. This will save you from duplicating the code above.